phpBB/includes/ucp/ucp_register.php

```diff
@@ -372,6 +372,22 @@ function main($id, $mode)
 
 				// Register user...
 				$user_id = user_add($user_row, $cp_data);
+				
+				/**
+				* Add into $user_row alter user_add 
+				*
+				* Add access to user_id 
+				*
+				* @event core.ucp_insert_user_row_after
+				* @var	bool	submit		Do we display the form only
+				*							or did the user press submit
+				* @var	array	cp_data		Array with custom profile fields data
+				* @var	array	user_row	Array with current ucp registration data
+				* @since 3.1.10-RC1
+				*/
+				$vars = array('submit', 'cp_data', 'user_row', 'user_id');
```

Comment:
At this point `$submit` will always be true, thus providing `$submit` will be senseless. (Also, the user has been created now, so.. what's the point of that variable, really?)